src/rpc/chain_api.rs

```diff
@@ -445,6 +447,33 @@ pub(crate) async fn chain_get_min_base_fee<DB: Blockstore>(
     Ok(min_base_fee.atto().to_string())
 }
 
+pub(crate) fn chain_notify<DB: Blockstore>(
+    _params: Params<'_>,
+    data: Arc<RPCState<DB>>,
+) -> Subscriber<ApiHeadChange> {
+    let (sender, receiver) = broadcast::channel(100);
+
+    // As soon as the channel is created, send the current tipset
+    let current = data.chain_store.heaviest_tipset();
+    let (change, headers) = ("current".into(), current.block_headers().clone().into());
+    let _ = sender.send(ApiHeadChange { change, headers });
```

Comment:
Let's assert that this call doesn't fail.

Reply:
Should the function return a `Result<Subscriber<ApiHeadChange>, JsonRpcError>`?
Or do you mean we should use a static assert?